### DISCUSSION TITLE
Visual Studio 2022 locks up, completely unresponsive if Copilot Settings menu is clicked prior to Plugin being activated

### DISCUSSION THREAD

**Discussion by wesmacdonald** [community/community#13883] [Copilot Conversations]
This is just feedback - plugin works for me as expected after activation.

I installed Copilot in VS 2022 (did not activate), restarted Visual Studio and did not open a project - I went into Edit | Copilot Suggestions | Settings... it locks up Visual Studio completely. Tried with VS 2022 (17.1.2) and the Preview (17.2.0-2.1) versions. 

![image](https://user-images.githubusercontent.com/1325956/160859010-1ca772b7-8d3b-4621-8eb0-0608845cfcdf.png)

Once the plugin is activated, the Settings dialog is displayed.

**Comment by kzu** [comment]
Should be fixed in the latest release (v1.10.0.2 or later). Please update and give it a try!

**Reply by wesmacdonald** [reply]
I just tried it again (latest plugin) with VS 2022 17.2.0-2.1 and I see the same behavior.

![image](https://user-images.githubusercontent.com/1325956/161039206-558fda24-b29b-4f9c-ab07-0b5840f372e3.png)

**Reply by kzu** [reply]
Ah, dang. The fix didn't make it to that build, but it's there in v1.10.0.3 and later. Please let me know if that fixed it for you! Thanks 💯

**Reply by wesmacdonald** [reply]
All fixed in 1.10.0.3, and it even kicks off the activation process! Nice work @kzu 

![image](https://user-images.githubusercontent.com/1325956/161318875-84e22047-c460-468d-9188-baf911631216.png)
